.clang-tidy

```
@@ -0,0 +1,3 @@
+Checks: '-*,readability-braces-around-statements,modernize-use-override'
```

For readability, I think you can do this:
```suggestion
Checks: >
  '-*,
  readability-braces-around-statements,
  modernize-use-override
```

That's what is done in https://github.com/libsdl-org/SDL/blob/main/.clang-tidy

---

Oh f me, I left a dash. Fixing it..